src/avram/errors.cr

```diff
@@ -30,6 +30,12 @@ module Avram
     end
   end
 
+  class MissingRequiredAssociationError < AvramError
+    def initialize(model : Avram::Model.class, association : Avram::Model.class)
+      super "Expected #{model} to have an association with #{association} but one was not found."
```

Comment:
Error message could be better. Any ideas?

Current:
<img width="858" alt="Screen Shot 2022-02-14 at 11 33 36 PM" src="https://user-images.githubusercontent.com/17329408/153998868-93bfccc3-c0fc-40dd-ace7-b1f08dd439e9.png">

Reply:
How about `Expected Comment to be assigned to a Post but no association was supplied`?

Or `Expected Comment to belong to a Post but no Post was supplied`

Reply:
Whoops, forgot to wait for feedback on this 😬 Feel free to open a PR with an error message that you think is better. Otherwise I _might_ remember to make one at some point

Reply:
Well. I could probably have also looked to see if it was merged before trying to be helpful...